core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html

```diff
@@ -128,6 +128,7 @@ <h4 class="title-table">Executors</h4>
               Shuffle Write</span></th>
           <th>Logs</th>
           <th>Thread Dump</th>
+          <th>Exec Loss Reason</th>
```

Comment:
Is this empty always in non-K8s resource managers?

Reply:
No exec loss reason is populated for YARN as well :)